src/genotyper.cc

```diff
@@ -108,15 +108,47 @@ class FormatFieldHelper : public IFormatFieldHelper {
         return ans;
     }
 
+    Status get_default_value(T* val) {
+        if (field_info.default_type == DefaultValueFiller::ZERO) {
+            *val = 0;
+        } else if (field_info.default_type == DefaultValueFiller::MISSING) {
+            switch (field_info.type) {
+                case RetainedFieldType::INT:
+                    *val = bcf_int32_missing;
+                    break;
+                case RetainedFieldType::FLOAT:
+                {
+                    float tmp = 0;
```

Comment:
`bcf_float_missing` (by value) is a `uint32_t` type, so `*val = bcf_float_missing` doesn't work... I would think that `bcf_float_set_missing(*val)` can do the job, but the compiler insist there's a type mismatch... This is the least verbose way that I could make it work 😦